### ISSUE TITLE
reproduce training dd

### ISSUE THREAD

**Issue by hppy139** [ddehun/DEnsity#4]
To reproduce training, can you share the training "tensor/" directory?
I am facing a overfitting situation that "train loss is close to zero".
![图片](https://github.com/user-attachments/assets/b7468110-7b93-4b3a-9f4f-722aff44d890)

**Comment by ddehun** [comment]
Hi. Thank you for your interest in our work.

Unfortunately, I recently left my lab by graduation, so I cannot access the tensor/ directory.

I have some questions to guess the problems.

1. Which hyperparameters do you use to train your reranker? I think https://github.com/ddehun/DEnsity/blob/master/scripts/train.sh this script can be used to obtain similar results to the paper.
2. It would be also good to evaluate your model in the downstream task (i.e., meta-evaluation dataset for dialogue).
3. If your targeted corpus is either ConvAI2 or DailyDialogue, You can also use the pre-released checkpoints in README (https://drive.google.com/drive/folders/1IUUg6xsmEr28oed2yPqIA2m6xsQ9yNRd). 

Please ping me if you have further issues. Thank you!

**Comment by hppy139** [comment]
And actually I have got your checkpoints models, I'm recently working on training on my own data("one round dialogue", i.e. speaker A one turn + speaker one turn). So, the first step for me is to reproduce your training.
    As for "1": yes, I am using the same parameters as this "DATASET=dd;  temp=0.1;  weight=1;  lr=5e-5".
    It's hard for me to distinguish which part got wrong...
(Bad to hear that "Unfortunately, I recently left my lab by graduation, so I cannot access the tensor/ directory." o(╥﹏╥)o)

**Comment by hppy139** [comment]
Besides, I'm wondering your time cost in training dd model. One epoch costs me 9.5 hours by using 4 Tesla K40c GPUs.
